src/schemas/22-0993/schema.js

```diff
@@ -0,0 +1,37 @@
+import _ from 'lodash';
+import definitions from '../../common/definitions';
+import schemaHelpers from '../../common/schema-helpers';
+
+let schema = {
+  $schema: 'http://json-schema.org/draft-04/schema#',
+  title: 'REQUEST TO OPT-OUT OF INFORMATION SHARING WITH EDUCATIONAL INSTITUTIONS (VA FORM 22-0993)',
+  type: 'object',
+  definitions: _.pick(definitions, [
+    'fullName',
+    'ssn',
+    'vaFileNumber',
+    'privacyAgreementAccepted'
```

Comment:
not needed, `schemaHelpers.addDefinitionToSchema` will add each definition, empty object should work for this field